Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -119,6 +119,10 @@ struct SeedFinderConfig {
   // which will make seeding very slow!
   float rMin = 33 * Acts::UnitConstants::mm;
 
+  // z of last layers to avoid iterations
+  float zMinLastLayer = -2800 * Acts::UnitConstants::mm;
+  float zMaxLastLayer = 2800 * Acts::UnitConstants::mm;
```

Comment:
Indeed this is very experiment dependent since it requires information on the geometry of the detector. So it is good to have it as a configurable parameter. 

I would though prefer to have clearer names for these parameters. 'zMinLastLayer' and `zMaxLastLayer` are quite puzzling